### PR TITLE
Kotlin: PFI Allowlist

### PR DESCRIPTION
**Pull request by angiejones**
## What type of PR is this? (check all applicable)

- [x] 📝 Documentation Update
- [x] 🧑‍💻 Code Snippet
- [x] 🧪 Tests

## Description

- Added Kotlin code snippet to determine if a PFI is part of the tbDEX network
- Added test for JS snippet
- Added test for Kotlin snippet
- Rearranged `test` arguments to run browser tests last

## Mobile & Desktop Screenshots/Recordings

image

## Added code snippets?
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [x] 🙅 no documentation needed

### CODE REVIEW

File: site/docs/tbdex/wallet/allowlist-pfis.mdx [hunk 1]
```diff
@@ -1,7 +1,14 @@
 ---
+title: Allowlisting PFIs
+hide_title: true
```
Comment (chris-tbd) [inline site/docs/tbdex/wallet/allowlist-pfis.mdx:3]: I think we want to show title, otherwise the language switcher goes above the title and it looks kind awkward

File: site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/wallet/PfiAllowListTest.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions.*
+
+/**
+ * Tests for Wallet AllowList guide
+ */
+class PfiAllowListTest {
+
+    @Test
+    fun `check if DID has PFI service`() {
```
Comment (chris-tbd) [inline site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/wallet/PfiAllowListTest.kt:19]: Do we have all this setup that we can use from another test file, and then just add this snippet in? Or do we have to duplicate this code?

**Review by chris-tbd — APPROVED** [review]

File: site/docs/tbdex/wallet/allowlist-pfis.mdx (hunk 1 above)
Comment (angiejones) [reply site/docs/tbdex/wallet/allowlist-pfis.mdx:3]: we want it above the title. you think it looks weird? i thought it looked weird inline with content

File: site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/wallet/PfiAllowListTest.kt (hunk 2 above)
Comment (angiejones) [reply site/testsuites/testsuite-kotlin/src/test/kotlin/docs/tbdex/wallet/PfiAllowListTest.kt:19]: i think we may be able to once @dayhaysoos adds his changes
